react/Condition.ts

```diff
@@ -54,7 +56,7 @@ const Condition: StorefrontFunctionComponent<ConditionProps> = ({
     return null
   }
 
-  return (children as any) ?? null
+  return (children as never) ?? null
```

Comment:
You could create this component as a normal function and type `children` the right way. Just a suggestion

Reply:
I prefer keeping it as a react component. The `never/any` workaround is needed because of out-of-date types for react in the `@types/react` package.